### ISSUE TITLE
Example of WebAPI CRUD using DnnAuthorize permisiosn

### ISSUE THREAD

**Issue by workcontrolgit** [OPSI-srl/DNN-9.x-SPA-and-Angular-6-7#2]
How does security work when Angular calling DNN WebAPI?  Is JWT supported?

The closest technical reference I could find is at http://www.bitboxx.net/Blog/Post/525/DNN-module-development-with-Angular-2-Part-7#/

If possible, please provide a simple CRUD example that works with DnnAuthorize

**Comment by joinsaturn** [comment]
yes security work with dnn webapi, tomorrow I’ll try to make an example and video....I do not use jwt for security because I use dnn security.

**Comment by joinsaturn** [comment]
I've made improvements to the github projects, maybe you'd better make a git pull.
Video: https://www.youtube.com/watch?v=OpX64e-oPjA

let me know...
